django/db/models/functions.py

```diff
@@ -83,6 +83,32 @@ def _paired(self, expressions):
         return ConcatPair(expressions[0], self._paired(expressions[1:]))
 
 
+class Greatest(Func):
+    """
+    Chooses the maximum non-null expression and returns it.
+    """
+    function = 'GREATEST'
+
+    def __init__(self, *expressions, **extra):
+        if len(expressions) < 1:
```

Comment:
Just make it:

```
if len(expressions) < 2:
```

That avoids problems with sqlite and mysql. GREATEST(x) is always x on backends that support single arguments anyway.